### PR TITLE
remove normalizing of splits

### LINKED ISSUE #2362: Height of splits is not accurately loaded
**Describe the bug**
When having multiple splits in the same column so under another the sizes of these splits don't get saved into the config very accurately upon closing Chatterino.

Edit: They get saved correctly, Chatterino loads them and then changes them a bit for some reason and saves that new value into the config.

Maybe window size in general isn't saved very accurate either, because I think the width of my chatterino isn't saving perfectly either. 

Filters are really useful and I use vertical splits a lot with them to have little mini chats below, that's why the accurate size is important to me. Currently I resize every time I start chatterino.

**To reproduce**
Create a tab with atleast 2 splits vertically. (So in 1 column)
Adjust the size for to a very small split and wait for the layout to save. Then close and reopen chatterino.

**Screenshots**
Before closing:
https://i.imgur.com/zMunuES.png

After Closing:
https://i.imgur.com/x3ubDTu.png

**Chatterino version**
Chatterino 2.2.3-beta2 (commit d0deb44ab)

**Operating system**
Manjaro 20.2.1 with i3

### PR DESCRIPTION
**Pull request by gempir**
Could not find any benefit of normalization, but splits got resized to bigger than they should. 

Fixes #2362

Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

I'm testing this on my machine for now, but I can't see any regressions with this and I have a pretty big split setup with like 3 columns and each having 3-4 rows

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -82,6 +82,7 @@
 - Bugfix: Fix CLI arguments (`--help`, `--version`, `--channels`) not being respected (#2368, #2190)
 - Bugfix: Fix Twitch cheer emotes not displaying tooltips when hovered (#2434)
 - Bugfix: Fix directory not opening when clicking "Open AppData Directory" setting button on macOS (#2531, #2537)
+- Bugfix: Fix splits beeing not accurately sized as they were when the application was previously open
```
Comment (leon-richardt) [inline CHANGELOG.md:85]:
```suggestion
- Bugfix: Fix splits being not accurately sized as they were when the application was previously open (#2362, #2548)
```

**Review by leon-richardt — COMMENTED** [review]
I only have input on the changelog entry unfortunately :man_shrugging:

**Comment by pajlada** [comment]
## Without this fix

**Before reopening chatterino**
2021-03-21-174810_640x480_scrot

**After reopening it once**
2021-03-21-174828_640x480_scrot

**After reopening it twice**
2021-03-21-174848_640x480_scrot

## With this fix

**Before reopening chatterino**
2021-03-21-175132_640x480_scrot

**After reopening it once**
2021-03-21-175142_640x480_scrot

**After reopening it twice**
2021-03-21-175150_640x480_scrot

There was a single change in the "width flex" here once after fixing but I can't reproduce it so that might've been me fatfingering the split size

TL;DR: Looks good to me!

**Review by pajlada — APPROVED** [review]

**Comment by fourtf** [comment]
3x

I recommend this fix instead: https://github.com/Chatterino/chatterino2/pull/2554

**Comment by pajlada** [comment]
> 3x
> 
> I recommend this fix instead: #2554

I will try it out in the same way I tried this build 
image

**Comment by gempir** [comment]
What do I need to do to get this merged? This PR seems to have less regressions than fourtf's and personally less code is more IMO.

**Comment by pajlada** [comment]
> What do I need to do to get this merged? This PR seems to have less regressions than fourtf's and personally less code is more IMO.

fourtf's PR #2554 has been updated and fixes all issues now, so I'll go ahead and close this issue.

Thanks for bringing the issue to our attention!
